app-misc/radeontop/radeontop-9999.ebuild

```diff
@@ -1,7 +1,7 @@
-# Copyright 1999-2017 Gentoo Foundation
```

Comment:
Don't add version to commit message prefix; make it match the directory instead. Instead, mention it in the summary.